### PR TITLE
Style Changes and Linter

### PR DESCRIPTION
**Pull request by superdiana**

### CODE REVIEW

**Comment by superdiana** [comment]
Style changes

**Review by slorello89 — CHANGES_REQUESTED** [review]
Looks like a couple optional parameters were added to the opentok constructor which would strike me as being outside of the scope of this PR. Also just one comment on some of the multi-line string decleartions. If the latter is considered the standard in python I'm fine with it, just strikes me as a bit odd (as they look like tuple declearations).

File: opentok/endpoints.py [hunk 1]
```diff
@@ -3,67 +3,94 @@ class Endpoints(object):
...
+            url += "/connection/" + connection_id
 
-        url += '/signal'
+        url += "/signal"
         return url
 
     def get_stream_url(self, session_id, stream_id=None):
         """ this method returns the url to get streams information """
-        url = self.api_url + '/v2/project/' + self.api_key + '/session/' + session_id + '/stream'
+        url = (
```
[the remark this answers is not in the document]
Comment (superdiana) [reply opentok/endpoints.py:32]: Not python blasphemy at all but the parenthesis should stay 🤓

File: opentok/opentok.py [hunk 2]
```diff
@@ -37,51 +37,69 @@
...
     """For internal use."""
 
-    def __init__(self, api_key, api_secret, api_url='https://api.opentok.com', timeout=None):
+    def __init__(
+        self,
+        api_key,
+        api_secret,
+        api_url="https://api.opentok.com",
+        timeout=None,
+        app_name="OpenTok-Python-SDK",
```
[the remark this answers is not in the document]
Comment (superdiana) [reply opentok/opentok.py:93]: It's the standard :v

[the remark this answers is not in the document]
Comment (dragonmantank) [reply opentok/opentok.py:93]: I don't believe the app name should be set by default, this is for app tracking. We shouldn't default on this  and it should be a purely user-supplied value.

We also have to be very careful about manipulating the user agent, as the OpenTok standards are different from ours at the moment, and they have a bunch of reports that work off their user agent formats.

File: tests/test_getter_setter.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        self.api_key = u("123456")
+        self.api_secret = u("1234567890abcdef1234567890abcdef1234567890")
+        self.session_id = u(
+            "1_MX4xMjM0NTZ-flNhdCBNYXIgMTUgMTQ6NDI6MjMgUERUIDIwMTR-MC40OTAxMzAyNX4"
+        )
+        self.opentok = OpenTok(self.api_key, self.api_secret)
+
+    def test_getset_appname(self):
+        assert self.opentok.app_name == "OpenTok-Python-SDK"
+        self.opentok.app_name = "Vonage-Video-SDK"
```
Comment (dragonmantank) [inline tests/test_getter_setter.py:20]: As mentioned in the constructor comment, we shouldn't set a default app name.

**Review by dragonmantank — CHANGES_REQUESTED** [review]
Just need to revert the app name change, and should be good!

File: opentok/__init__.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
     ForceDisconnectError,
     ArchiveError,
     SetStreamClassError,
-    BroadcastError
+    BroadcastError,
```
[the remark this answers is not in the document]
Comment (superdiana) [reply opentok/__init__.py]: That's black actually steve

File: tests/test_getter_setter.py (hunk 3 above)
Comment (superdiana) [reply tests/test_getter_setter.py]: @dragonmantank 
- Removed default value assignment from the constructor
- Removed getters and setters for `app_name`
- Removed getters and setters test for `app_name`
- Executed tests: `nosetests -v tests/test_*` with successful output

**Review by dragonmantank — APPROVED** [review]

**Pull request merged**